import_data/tasks/tasks.py

```diff
@@ -750,14 +753,24 @@ def read_osm_timestamp(ctx, osm_file_path):
 
 @task
 @format_stdout
-def init_osm_update(ctx):
+def init_osm_update(ctx, osm_file_path=None):
```

Comment:
@sinaure You're right, the decorator `@format_stdout` breaks how `invoke` parses the arguments.
See https://github.com/pyinvoke/invoke/issues/555

An alternative would be to use `INVOKE_OSM_FILE` env var to override the `ctx.osm.file` value.